packages/common/src/constants/regex.ts

```diff
@@ -0,0 +1,2 @@
+export const PRELOAD_UID_PATTERN =
+	/^uuid:[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$/;
```

Comment:
This module is named generally, because I anticipate at least a few other patterns moving here.

Reply:
Not related to edit, at least that I'm aware of. I would probably have moved those now if I expected to use them right away.

A few that come to mind:

- [XML/XPath names](https://github.com/getodk/web-forms/blob/cbfe1d3480c2e1a3d3f6785f14d86e67aace7ec6/packages/tree-sitter-xpath/src/grammar.ts#L25-L44)
- [Date/time](https://github.com/getodk/web-forms/blob/cbfe1d3480c2e1a3d3f6785f14d86e67aace7ec6/packages/xpath/src/functions/xforms/datetime.ts#L246-L249) (I think you've linked others at some point as well)

There may be others that aren't bubbling up in my memory, my recall isn't exactly at its best right now!

Reply:
FYI note: I’ve extracted the regex for [date time](https://github.com/getodk/web-forms/pull/328) into its own file, since it’s substantial enough to stand alone rather than being combined with other regex patterns in a single file